WorkloadManagementSystem/Utilities/JobParameters.py

```diff
@@ -19,14 +20,16 @@ def getJobFeatures():
     features[item] = val
   return features
 
+def getProcessorFromMJF():
+  features = getJobFeatures()
+  NumberOfProcessor = features.get( 'allocated_cpu' )
+  return NumberOfProcessor
```

Comment:
Can be a single line. Then also the naming convention ( numberOfProcessors ) would not be violated